google-dracut-config.spec

```diff
@@ -34,28 +34,29 @@ BuildArch:      noarch
 %build
 
 %install
-install -D -m 644 etc/dracut.conf.d/07-dracut-nvme.conf \
-    %{buildroot}%{_sysconfdir}/dracut.conf.d/07-dracut-nvme.conf
-install -D -m 644 etc/dracut.conf.d/07-ext4.conf \
-    %{buildroot}%{_sysconfdir}/dracut.conf.d/07-ext4.conf
-install -D -m 644 etc/dracut.conf.d/07-virtio.conf \
-    %{buildroot}%{_sysconfdir}/dracut.conf.d/07-virtio.conf
-install -D -m 644 etc/dracut.conf.d/07-xfs.conf \
-    %{buildroot}%{_sysconfdir}/dracut.conf.d/07-xfs.conf
-install -D -m 644 etc/dracut.conf.d/11-resume.conf \
-    %{buildroot}%{_sysconfdir}/dracut.conf.d/11-resume.conf
+install -D -m 644 usr/lib/dracut/dracut.conf.d/07-dracut-nvme.conf \
+    %{buildroot}/usr/lib/dracut/dracut.conf.d/07-dracut-nvme.conf
+install -D -m 644 usr/lib/dracut/dracut.conf.d/07-ext4.conf \
+    %{buildroot}/usr/lib/dracut/dracut.conf.d/07-ext4.conf
+install -D -m 644 usr/lib/dracut/dracut.conf.d/07-virtio.conf \
+    %{buildroot}/usr/lib/dracut/dracut.conf.d/07-virtio.conf
+install -D -m 644 usr/lib/dracut/dracut.conf.d/07-xfs.conf \
+    %{buildroot}/usr/lib/dracut/dracut.conf.d/07-xfs.conf
+install -D -m 644 usr/lib/dracut/dracut.conf.d/11-resume.conf \
+    %{buildroot}/usr/lib/dracut/dracut.conf.d/11-resume.conf
 
 %check
 
 %files
 %doc README.md
 %license LICENSE
-%dir %{_sysconfdir}/dracut.conf.d/
-%config %{_sysconfdir}/dracut.conf.d/07-dracut-nvme.conf
-%config %{_sysconfdir}/dracut.conf.d/07-ext4.conf
-%config %{_sysconfdir}/dracut.conf.d/07-virtio.conf
-%config %{_sysconfdir}/dracut.conf.d/07-xfs.conf
-%config %{_sysconfdir}/dracut.conf.d/11-resume.conf
+%dir /usr/lib/dracut/
+%dir /usr/lib/dracut/dracut.conf.d/
+%config /usr/lib/dracut/dracut.conf.d/07-dracut-nvme.conf
+%config /usr/lib/dracut/dracut.conf.d/07-ext4.conf
+%config /usr/lib/dracut/dracut.conf.d/07-virtio.conf
+%config /usr/lib/dracut/dracut.conf.d/07-xfs.conf
+%config /usr/lib/dracut/dracut.conf.d/11-resume.conf
```

Comment:
Those files shouldn't be marked as `%config`  as they are not intended to be altered.